WordPress/src/main/java/org/wordpress/android/ui/reader/discover/viewholders/ReaderPostViewHolder.kt

```diff
@@ -58,6 +62,7 @@ class ReaderPostViewHolder(
         )
         uiState.postHeaderClickData?.onPostHeaderViewClicked?.let {
             layout_post_header.setOnClickListener {
+                AnalyticsTracker.track(READER_SITE_SHOWN)
```

Comment:
We do most of the tracking within the VM is there any reason why we want to track these two cases here (view layer)? Tbh I wouldn't mind if everything would be in the view layer but it feels a bit inconsistent now. Wdyt?

Reply:
@malinajirka - I dropped it here because it covers all of reader, not just a particular view model, so we don't inadvertently miss a track.